scene/main/http_request.cpp

```diff
@@ -154,6 +154,10 @@ Error HTTPRequest::request_raw(const String &p_url, const Vector<String> &p_cust
 		err = _request();
 		if (err != OK) {
 			call_deferred(SNAME("_request_done"), RESULT_CANT_CONNECT, 0, PackedStringArray(), PackedByteArray());
+			// Maximum number of DNS queries reached.
+			if (err == ERR_BUSY) {
+				return ERR_BUSY;
+			}
```

Comment:
I don't think this is needed.

Reply:
I wanted to return a more unique error to notify that this was happening but it being more generic wouldn't hurt, I suppose?